examples/foe-write.rs

```diff
@@ -0,0 +1,27 @@
+use std::io::Read;
+
+use ethercat::{Master, MasterAccess};
+
+fn main() -> Result<(), std::io::Error> {
+    let mut master = Master::open(0, MasterAccess::ReadWrite)?;
+
+    let args: Vec<String> = std::env::args().collect();
+    if args.len() < 4 {
+        eprintln!("Usage: foe-read <slave-position> <foe-name> <file>");
+        return Err(std::io::Error::new(
+            std::io::ErrorKind::Other,
+            "Not enough arguments",
+        ));
+    }
+    let slave_idx = args[1]
+        .parse::<u16>()
+        .map_err(|e| std::io::Error::new(std::io::ErrorKind::Other, e))?;
+    let foe_name = &args[2];
+    let file = &args[3];
+    let mut fd = std::fs::OpenOptions::new().read(true).open(file)?;
```

Comment:
This is just `std::fs::read()`?

Reply:
Oh yeah, I'm somewhat used to `OpenOptions`, forgot the shortcut